proof/refine/X64/Detype_R.thy

```diff
@@ -727,92 +591,95 @@ lemma sym_refs_ko_wp_atD:
 lemma zobj_refs_capRange:
   "capAligned c \<Longrightarrow> zobj_refs' c \<subseteq> capRange c"
   by (cases c, simp_all add: capRange_def capAligned_def is_aligned_no_overflow)
+
 end
+
 locale delete_locale =
-  fixes s and base and bits and ptr and idx and d
-  assumes cap: "cte_wp_at' (\<lambda>cte. cteCap cte = UntypedCap d base bits idx) ptr s"
-  and  nodesc: "descendants_range' (UntypedCap d base bits idx) ptr (ctes_of s)"
-  and    invs: "invs' s"
-  and  ct_act: "ct_active' s"
-  and sa_simp: "sch_act_simple s"
-  and     bwb: "bits < word_bits"
+  fixes s' and base and bits and ptr and idx and d
```

Comment:
is there any reason the delete_locale got a rename of `s` to `s'`? this generated a big diff

Reply:
I think it was actually unnecessary, but I think I did this for MCS originally because it helped me keep track of which states are abstract and which are concrete: we have the `detype_locale'` here which has both abstract and concrete states, and then the `delete_locale` (which I modify here) which has just the concrete state. I can revert this, but I should point out that it's like this on the `rt` branch

Reply:
No no, please keep it, it's better when things make intuitive sense. It did create a bit of diff noise, so I wanted to know why.

Reply:
I was about to ask the same question. Thanks for the explanation, also happy to leave as is.